lib/modules/encounters.rb

```diff
@@ -38,7 +38,7 @@ class Encounters < Synthea::Rules
                   6.months
               end
             end
-            next_date = time + Distribution::Normal.rng(delta, delta*schedule_variance).call
+            next_date = time + Distribution::Normal.rng(delta*1.0, delta*schedule_variance).call
```

Comment:
Without the 1.0 multiplier this throws an error in JRuby. Normal.rng expects a float and when it gets something else it can't find the right function to call so it errors out. I didn't try any other ways to convert to a float so there may be something more intuitive.